lib/ExternalModule.js

```diff
@@ -28,6 +28,17 @@ const makeSerializable = require("./util/makeSerializable");
 /** @typedef {import("./util/Hash")} Hash */
 /** @typedef {import("./util/fs").InputFileSystem} InputFileSystem */
 
+/**
+ * @param {string} strPath the raw path e.g "my-lib/alpha/beta/gamma"
+ * @returns {string} the generated array chain '["my-lib"]["alpha"]["beta"][gamma]'
+ */
+const pathChain = strPath => {
+	return strPath
+		.split("/")
+		.map(chainLink => `["${chainLink}"]`)
```

Comment:
There is a little helper function which can create these accessors: `lib/util/propertyAccess`.

Could you use this one instead? It's a little more reliable and pretty for some edge-cases like special chars in the external. e. g. `import "ext-lib/0alpha/be\"ta/gamma"`. Maybe you could change your test case to these complex case?

Reply:
Good spot, I will use it